interface/ceed-qfunction.c

```diff
@@ -309,13 +309,14 @@ int CeedQFunctionGetFortranStatus(CeedQFunction qf, bool *fortranstatus) {
   @ref Advanced
 **/
 
-int CeedQFunctionGetFortranContext(CeedQFunction qf, void* *ctx) {
-  if (!qf->fortranstatus)
-    return CeedError(qf->ceed, 1,
-                     "QFunction was not set using Fortran");
+int CeedQFunctionGetInnerContext(CeedQFunction qf, void* *ctx) {
+  if (qf->fortranstatus) {
+    fContext *fctx = qf->ctx;
+    *ctx = fctx->innerctx;
```

Comment:
Fine for now, but more extensible (possibly needed in the future) would be to have a method (function pointer) that is called if it exists so that this if statement doesn't possibly need to be built out to handle future cases.